packages/ses/src/lockdown.js

```diff
@@ -408,6 +382,12 @@ export const repairIntrinsics = (options = {}) => {
     markVirtualizedNativeFunction,
   });
 
+  if (legacyHermesTaming === 'unsafe') {
```

Comment:
```suggestion
  if (legacyHermesTaming === 'safe') {
```

i.e. disable Compartment by default